02/app.js

```diff
@@ -0,0 +1,13 @@
+const formEl = document.querySelector('form');
+console.log(formEl);
+formEl.addEventListener('submit', function(e)
+{
+    e.preventDefault();
+})
+
+const handleChange = function(e) {
+        console.log(        e.target.name,        e.target.value,    );
+     } 
+for(const el of formEl.elements)
+ {    el.addEventListener('change', handleChange);   
+  }
```

Comment:
Ten kod może być, ale nie musi. Chodzi o to kiedy chcesz sprawdzać poprawność danych. Czy już na etapie ich wprowadzania czy dopiero po wysłaniu. Z zadania wynika raczej, że powinieneś to zrobić na event o nazwie `submit` więc `change` tutaj jest niepotrzebne.